### DISCUSSION TITLE
Is there a table component?

### DISCUSSION THREAD

**Discussion by transitive-bullshit** [primer/react#1974] [Q&A] [answered]
Comparing different component libraries, I was considering primer because I love the github feel and the library seems solid, but it doesn't look like it has a built-in way to display a table of data?

This seems like "table stakes" (hehehe) when choosing a component library these days (ala chakra, antd, material-ui, etc)

Any word on if this is on the roadmap and/or the best way to display tabular data with `@primer/react`?

Related: https://github.com/primer/react/issues/708

**Comment by lesliecdubs** [comment, marked as answer]
Hi there 👋🏻 A `DataTable` Primer component is currently in design and will likely be moving to engineering in the new year. Stay tuned!

**Reply by transitive-bullshit** [reply]
Awesome news. Thanks @lesliecdubs 🙏
